test/widgets/login_test.dart

```diff
@@ -61,7 +64,44 @@ void main() {
     expectErrorFromText('email@example.com', ServerUrlValidationError.noUseEmail);
   });
 
-  // TODO test AddAccountPage
+  group('AddAccountPage server URL helper text', () {
+    Future<void> prepareAddAccountPage(WidgetTester tester) async {
+      await tester.pumpWidget(const MaterialApp(
+        localizationsDelegates: ZulipLocalizations.localizationsDelegates,
+        supportedLocales: ZulipLocalizations.supportedLocales,
+        home: AddAccountPage(),
+      ));
+    }
+
+    final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
```

Comment:
This `GlobalLocalizations.zulipLocalizations` is application state that can change. So reading it should happen within individual test cases (including helpers they call), i.e. individual `testWidgets` callbacks, not outside them.